src/disco/containers/InstallButton.js

```diff
@@ -143,9 +152,11 @@ export function mapDispatchToProps(dispatch) {
       });
     },
 
-    uninstall({ guid, installURL, slug }) {
+    uninstall({ guid, installURL, slug, name, type }) {
       const addonManager = new AddonManager(guid, installURL);
       dispatch({type: 'START_UNINSTALL', payload: {slug}});
+      const action = type === THEME_TYPE ? 'theme' : 'addon';
+      tracking.sendEvent({action, category: UNINSTALL_CATEGORY, label: name});
```

Comment:
Heh, it's not so much the ternary as much as it is that not being a theme doesn't mean that it is an add-on. Ternaries just happen to be where it's likely for something like that to happen.